lib/commands/config/index.js

```diff
@@ -119,7 +121,7 @@ class ConfigCommand extends Command {
                     type: 'input',
                     name: 'dbname',
                     message: 'Enter your Ghost database name:',
-                    default: this.instance.config.get('database.connection.database', `ghost_${this.system.environment}`)
+                    default: this.instance.config.get('database.connection.database', `ghost_${this.system.environment}_${path.resolve(process.cwd()).substr(path.resolve(process.cwd()).lastIndexOf('/') + 1)}`)
```

Comment:
instead of the `path.resolve` stuff, you can just do `path.basename(process.cwd())` and it will get you the last section of the path 😁